.babelrc

```diff
@@ -1,9 +1,18 @@
 {
-  "plugins": [
-    "emotion",
-    "transform-class-properties",
-    "transform-object-rest-spread"
+  "plugins": ["emotion", "@babel/plugin-proposal-class-properties"],
+  "presets": [
+    ["@babel/preset-env", { "modules": false }],
+    "@babel/preset-react",
+    "@babel/preset-flow"
   ],
-  "presets": ["env", "react"],
-  "ignore": ["node_modules"]
+  "ignore": ["node_modules"],
+  "env": {
```

Comment:
If the answer to the above query is no, then we can remove this additional configuration